hw/bus/src/bus.c

```diff
@@ -285,8 +292,18 @@ bus_node_unlock(struct os_dev *node)
     return 0;
 }
 
+os_time_t
+bus_node_get_lock_timeout(struct os_dev *node)
```

Comment:
I assume this is not the final implementation, and that bus_node_set_lock_timeout() is going to follow?

Reply:
my idea here is to let user specify per-node lock timeout in node configuration and getter is to return effective lock timeout for a node, i.e. either default one (if not configured) or the one configured for a node. I think there's no reason for user to reconfigure lock timeout in runtime - by default all nodes can be locked with the same /reasonable/ timeout but perhaps some nodes should be accessed with e.g. very short lock timeout so they either can be read instantly or not at all and for this it's enough to configure this on build time. but there may be some actual scenarios which require different behaviour so comments are always welcome :)

I'll include implementation of this in next series of commits I'm working on currently.